portal-impl/src/portal.properties

```diff
@@ -2104,16 +2104,9 @@
     users.reminder.queries.questions=what-is-your-primary-frequent-flyer-number,what-is-your-library-card-number,what-was-your-first-phone-number,what-was-your-first-teacher's-name,what-is-your-father's-middle-name
 
     #
-    # Set this to false to disable the user indexer.
-    #
-    users.indexer.enabled=true
-
-    #
     # Set this to true to search users from the index. Set this to false to
     # search users from the database. Note that setting this to false will
-    # disable the ability to search users based on Expando attributes. This
-    # setting is not used unless the property "users.indexer.enabled" is set
-    # to true.
+    # disable the ability to search users based on Expando attributes.
```

Comment:
Instead of removing this warning completely, should we change the reference to the obsolete property and point the user to the UI?